File: sdkwork-ai-sdk-go/types/chat_completion_list.go
package types

// OpenAI标准聊天完成列表
type ChatCompletionList struct {
	Object string `json:"object"`
	Data []ChatCompletionResponse `json:"data"`
	HasMore bool `json:"has_more"`
	FirstId string `json:"first_id"`
	LastId string `json:"last_id"`
}

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/ChatCompletionList.kt
package com.sdkwork.ai

data class ChatCompletionList(
    val object: String? = null,
    val data: List<ChatCompletionResponse>? = null,
    val hasMore: Boolean? = null,
    val firstId: String? = null,
    val lastId: String? = null
)
